selenium-grid-setup.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/bash
+
+export PBHOSTURL
```

Comment:
MPSUPPLIERURL ?

Reply:
This needs to be PBHOSTURL to work with the hosts file correctly